experimental/traffic-portal/src/app/api/export-attachment.service.ts

```diff
@@ -0,0 +1,50 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { HttpClient } from "@angular/common/http";
+import { Injectable } from "@angular/core";
+import { ProfileExport, ResponseProfile } from "trafficops-types";
+
+import { environment } from "src/environments/environment";
+
+/**
+ * Defines & handles api endpoints related to export attachments/json/text
+ * Here we are not using the base-api-service to tap the response.
+ */
+@Injectable()
+export class ExportAttachmentService {
```

Comment:
I don't think we need a new service for this, I think exporting a profile should be in the Profile Service, which is also where you put Profile imports, so I think they ought to go together.

Reply:
its a text editor only, have developed a seperate reusable component for future use of silimar use cases. I think we need to get it discussed on a. huddle.